### PR TITLE
Srinivas8350/sc 296/kyraa candy machine

### PR DESCRIPTION
**Pull request by valekar**

### CODE REVIEW

File: packages/candy-machine/package.json [hunk 1]
```diff
@@ -0,0 +1,64 @@
+{
+  "name": "candy-machine",
```
Comment (shivekkhurana) [inline packages/candy-machine/package.json:2]: This should be `@kyraa/candy-machine`

File: packages/candy-machine/src/connection.ts [hunk 2]
```diff
@@ -0,0 +1,324 @@
...
+            resolve(status);
+          }
+        },
+        commitment
+      );
+    } catch (e) {
+      done = true;
+      log.error('WS error in setup', txid, e);
+    }
+    while (!done && queryStatus) {
```
Comment (shivekkhurana) [inline packages/candy-machine/src/connection.ts:285]: What is this sorcery ?

**Review by shivekkhurana — APPROVED** [review]
Can we add a basic Readme ?

File: packages/candy-machine/src/connection.ts (hunk 2 above)
Comment (valekar) [reply packages/candy-machine/src/connection.ts:285]: This verifies if the signatures are confirmed, it's like retry mechanism. 

We could use this retry mechanism as well.

**Pull request merged**